radio/apps/schedules/views.py

```diff
@@ -49,11 +49,10 @@ def feed_schedules(request):
     )
 
 
-background_colours = {"L": "#F9AD81", "B": "#C4DF9B", "S": "#8493CA"}
-text_colours = {"L": "black", "B": "black", "S": "black"}
-
-
 def __get_events(after, before, json_mode=False):
+    background_colours = {"L": "#F9AD81", "B": "#C4DF9B", "S": "#8493CA"}
+    text_colours = {"L": "black", "B": "black", "S": "black"}
+
```

Comment:
background_colours and text_colours are local variables, no need to poison global variable space.